### DISCUSSION TITLE
Rotary encoder differences

### DISCUSSION THREAD

**Discussion by CaitSith2** [sanni/cartreader#280] [Coding Corner]
https://github.com/sanni/cartreader/blob/a675af91e112b076afd818f8da829124fd766f42/Cart_Reader/Cart_Reader.ino#L1101

I noticed while looking through the Rotary encoder version of the button holding, that there is no event 4,  super long hold,  which without this event, it is impossible to reset the dumping counter, which I figured out was done by long holding the left button while the cursor was on the About selection.

**Comment by sanni** [comment]
Good catch. 😃 
Added a 10 second button press in the about screen to reset folders.
